lib/core/src/server/templates/base-manager-head.html

```diff
@@ -0,0 +1,16 @@
+<style>
+  html, body {
+    overflow: hidden;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+</style>
+
+<script>
+  if (window.parent !== window) {
+    window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = window.parent.__REACT_DEVTOOLS_GLOBAL_HOOK__;
+    window.__VUE_DEVTOOLS_GLOBAL_HOOK__ = window.parent.__VUE_DEVTOOLS_GLOBAL_HOOK__;
```

Comment:
I think we have these both in globals. Anyway it's strange to have them together 🤔

Reply:
See https://github.com/storybooks/storybook/issues/102#issuecomment-408224573